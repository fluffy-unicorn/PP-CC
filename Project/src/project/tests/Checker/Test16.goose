# Correct usage of arrays

int[] as;
function void main() {
    as = {1,2,3,4};
    printArray(as);
}

function void printArray(int[] as) {
    int i = 0;
    while (i < len as) {
        print as[i];
        i = i + 1;
    }
}
