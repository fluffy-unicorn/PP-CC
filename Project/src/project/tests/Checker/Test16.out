Sprockell 0 says 1
Sprockell 0 says 2
Sprockell 0 says 3
Sprockell 0 says 4
